Initial project generation

hisaac opened the pull request:
This pull request adds the sample app generated by the Flutter CLI to the repo, as well as a basic `bitrise.yml` file for demonstration purposes.

hisaac committed:
Initial project generation

hisaac committed:
Add Bitrise-generated bitrise.yml file

hisaac committed:
Add trigger map

hisaac committed:
Allow flutter to update during build

ofalvai commented on bitrise.yml:
```diff
@@ -0,0 +1,70 @@
+
+meta:
+  bitrise.io:
+    stack: osx-xcode-13.2.x
```
Same concern as in the other repo

hisaac replied:
This has been removed now 👍

hisaac committed:
Remove stack/machine definition from bitrise.yml file

The stack and machine are overridden during the process of creating the sample app during onboarding, so there's no need to define them in the file within the repo. This may change later if we determine there's value in having it in the file, even though it's overridden.
